.github/workflows/build_binaries.yml

```
@@ -624,13 +638,13 @@ jobs:
           path: "${{ github.workspace }}/buildtools/Output/*"
 
       - name: Audit tree and feedback for binaries
-        # if: ${{ ( ! matrix.builds.cross ) }}
+        if: ${{ ( ! matrix.builds.cross ) }}
         continue-on-error: true
         shell: bash
+        working-directory: ${{ env.MTS_SOURCE }}
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2035:info:2:17: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)